runtime/doc/builtin.txt

```diff
@@ -6055,6 +6059,43 @@ matcharg({nr})							*matcharg()*
 		Can also be used as a |method|: >
 			GetMatch()->matcharg()
 
+matchbufline({buf}, {pat}, {lnum} [, {dict}])		*matchbufline()*
+		Returns the |List| of matches in line {lnum} in buffer {buf}
+		where {pat} matches.
```

Comment:
Currently, it is not documented what happens when `matchbufline()` is invoked on a buffer that hasn't been loaded yet (or a invalid buffer number is passed). Shouldn't the help be similar to `:h setbufline()`?

This is from `:h setbufline()`:
```
This function works only for loaded buffers. First call
|bufload()| if needed.

...

When {buf} is not a valid buffer, the buffer is not loaded or
{lnum} is not valid then 1 is returned.  In |Vim9| script an
error is given.
```

Reply:
Yes.  This should be similar to the setbufline() function.  I have updated the code, test and the help text for an unloaded buffer.